starport_template/.fvm/fvm_config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "flutterSdkVersion": "2.8.0",
+  "flutterSdkVersion": "2.10.1",
```

Comment:
could we also update the version in all other packages and run `fvm flutter pub update` there?

Reply:
sure we can

Reply:
Should we use `pub get upgrade` or just `pub get`, the former may break the codebase by updating to the latest versions of packages with breaking changes.

Reply:
let's go with `pub upgrade`. it should update only minor version of the libraries, which should be safe. Afterwards we can click trough the template and spot the eventual bugs, but I don't anticipate any

Reply:
No problem then.

Reply:
let me clean this up abit

Reply:
Here it is
<img width="561" alt="Screenshot 2022-02-17 at 11 12 08 AM" src="https://user-images.githubusercontent.com/25801929/154454401-66e10a2a-a115-4a41-9e11-edd1fd00754b.png">